grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -126,22 +126,72 @@ public GrpcServiceBuilder addService(ServerServiceDefinition service) {
         return this;
     }
 
+    /**
+     * Adds a gRPC {@link ServerServiceDefinition} to this {@link GrpcServiceBuilder}, such as
+     * what's returned by {@link BindableService#bindService()}.
+     *
+     * <p>Note that the specified {@code path} replaces the normal gRPC service path.
+     * Let's say you have the following gRPC service definition.
+     * <pre>{@code
+     * package example.grpc.hello;
+     *
+     * service HelloService {
+     *   rpc Hello (HelloRequest) returns (HelloReply) {}
+     * }}</pre>
+     * The normal gRPC service path for the {@code Hello} method is {@code "/example.grpc.hello/Hello"}.
+     * However if you set the {@code path} to {@code "foo"}, the {@code Hello} method will be served at
+     * {@code "/foo/Hello"}. This is useful for supporting unframed gRPC with HTTP idiomatic path.
+     */
+    public GrpcServiceBuilder addService(String path, ServerServiceDefinition service) {
```

Comment:
Oops realized I agreed with this idea in the issue but not after looking closer. I think this should take a `MethodDescriptor` too so the path can refer to a specific method. Would allow "/hello" to point to the Hello method. "/foo/Hello" still isn't so idiomatic

Reply:
I see. By the way, is `MethodDescriptor` too lower level for users? How about adding an option for method naming strategy such camelCase, snake_case, directory/style?

Reply:
Yeah it's an interesting point. But since `MethodDescriptor` is type safe, it seems both nice and direct as well as less chance of bugs, and I guess less cognitive load? It's not intuitive to me what sort of system we'd provide for renaming without thinking more deeply, so maybe the end result also won't be able intuitive. Not sure though.

Reply:
> But since MethodDescriptor is type safe, it seems both nice and direct as well as less chance of bugs, and I guess less cognitive load?

Agreed. A specific `MethodDescriptor` would be better than an ambiguous conversion.
Let me address it. 😀